src/frontend/src/components/ProposedSolutionForm.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import PageBlock from '../layouts/PageBlock';
+import { Button, Col, Form, InputGroup, Row } from 'react-bootstrap';
+import * as yup from 'yup';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+interface ProposedSolution {
+  description: string;
+  budgetImpact: number;
+  timelineImpact: number;
+  scope: string;
+}
+
+interface ProposedSolutionFormProps {
+  description?: string;
+  budgetImpact?: number;
+  timelineImpact?: number;
+  scope?: string;
+  readOnly?: boolean;
+  onAdd: (data: ProposedSolution) => void;
+}
+
+const schema = yup.object().shape({
+  description: yup.string().required('Description is required'),
+  budgetImpact: yup
+    .number()
+    .typeError('Budget Impact must be a number')
+    .min(0, 'Budget Impact must be greater than or equal to $0')
+    .required('Budget Impact is required')
+    .integer('Budget Impact must be an integer'),
+  scope: yup.string().required('Scope is required'),
+  timelineImpact: yup
+    .number()
+    .typeError('Timeline must be a number')
+    .min(0, 'Timeline must be greater than or equal to 0 weeks')
+    .required('Timeline is required')
+    .integer('Timeline must be an integer')
+});
+
+const styles = {
+  numberInput: {
+    width: '47.5%'
+  },
+  descColumn: {
+    paddingRight: 0
+  },
+  numberColumn: {
+    paddingLeft: 0
+  }
+};
+
+const ProposedSolutionForm: React.FC<ProposedSolutionFormProps> = ({
+  description,
+  budgetImpact,
+  timelineImpact,
+  scope,
+  readOnly,
+  onAdd
+}) => {
+  const { register, formState, handleSubmit } = useForm<ProposedSolution>({
+    resolver: yupResolver(schema),
+    defaultValues: { description, budgetImpact, timelineImpact, scope }
+  });
+
+  return (
+    <>
+      <PageBlock title={`Proposed Solution${readOnly ? ' - Read Only' : ''}`}>
```

Comment:
Remove Proposed Solution hardcoding -> component should just be the form and the optional add button, no tittle